ofmeet/src/java/org/jivesoftware/openfire/plugin/ofmeet/ConfigServlet.java

```diff
@@ -70,9 +79,11 @@ public void doGet( HttpServletRequest request, HttpServletResponse response ) th
 
             final OFMeetConfig ofMeetConfig = new OFMeetConfig();
 
+			final boolean isSwitchAvailable = JiveGlobals.getBooleanProperty("freeswitch.enabled", false);
             final String xmppDomain = XMPPServer.getInstance().getServerInfo().getXMPPDomain();
+            final String sipDomain = JiveGlobals.getProperty("freeswitch.sip.hostname", getIpAddress());
 
-            final JSONArray conferences = new JSONArray();
+            final JSONObject conferences = new JSONObject();
```

Comment:
What's the purpose of the `conferences` variable? I see it being passed to `handleAuthenticatedUser` where it gets filled, but I can't find where the values are being used.